backend/src/database/models/quiz.ts

```diff
@@ -51,6 +51,13 @@ export class Quiz extends Model {
   })
   organisation!: string
 
+  @Column({
+    type: DataType.STRING,
```

Comment:
Considering ULID has a fixed maximum length, we could set it:
```suggestion
    type: DataType.STRING(26),
```
(otherwise the default is 255.